css-ui-4/Overview.bs

```diff
@@ -2199,6 +2207,24 @@ Switching appearance: the 'appearance' property</h3>
 
 			Elements other than <a>widgets</a> must be rendered as for ''appearance/none''.
 
+		<dt><dfn>base</dfn>
+		<dd>
+			The effect of ''appearance/base'' depends on the element it is applied to:
+			<dl class=switch>
```

Comment:
Running bikeshed with this dl here, even without any further content, gives me the following error:
```
jarhar@jarhar-mac ~/csswg-drafts/css-ui-4 % pipx run bikeshed spec Overview.bs                             
LINE 2344: 'property' definitions don't use a 'for' attribute, but this one claims it's for 'appearance' (perhaps inherited from an ancestor). This is probably a markup error.
<dfn bs-line-number="2344" data-export="" data-dfn-type="property">-webkit-appearance</dfn>
FATAL ERROR: 'property' definitions don't use a 'for' attribute, but this one claims it's for 'appearance' (perhaps inherited from an ancestor). This is probably a markup error.
<dfn data-dfn-type="property">field-sizing</dfn>                                                                              
FATAL ERROR: 'property' definitions don't use a 'for' attribute, but this one claims it's for 'appearance' (perhaps inherited from an ancestor). This is probably a markup error.
<dfn data-dfn-type="property">input-security</dfn>                                                                            
```
It seems totally unrelated but doesn't happen without this dl. Am I missing something?

Reply:
I think you're missing `</dl>`.

Reply:
That fixed it, thanks!